src/ImGui/Desktop/Fusee.ImGui.Desktop/Templates/ImGuiFolderPicker.cs

```diff
@@ -20,6 +18,16 @@ public class ImGuiFolderPicker
         /// </summary>
         public EventHandler? OnCancel;
 
+        /// <summary>
+        /// Allow resizing of file picker window
+        /// </summary>
+        public bool AllowFilePickerResize { get; set; } = true;
```

Comment:
Check for naming continutity: "file" vs "folder" (throughout the complete file).

Reply:
Thx. Fixed.

Reply:
Sorry, what I meant with "throughout the complete file" was, that there are a few more variable containing "file" instead of folder.

Reply:
Should be fixed with newest push.